packages/strapi-helper-plugin/lib/src/components/InputAddonWithErrors/index.js

```diff
@@ -130,7 +130,12 @@ class InputAddonWithErrors extends React.Component {
         <InputDescription
           className={inputDescriptionClassName}
           message={inputDescription}
-          style={inputDescriptionStyle}
+          style={{
+            marginTop: '1.1rem',
+            marginBottom: '0.3rem',
+            lineHeight: 'normal',
```

Comment:
Yes because this component is from the helper-plugin so adjusting its style directly in the component will impact the other views where it is used plus the design rule where it is used is a bit different than a simple form so I agree with @virginieky : it is the best way to do it.

Reply:
Yes that will also work!